{
    "dataType": "CVE_RECORD",
    "cveMetadata": {
        "state": "PUBLISHED",
        "cveId": "CVE-2024-33663",
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "dateUpdated": "2024-09-03T19:34:19.749577",
        "dateReserved": "2024-04-25T00:00:00",
        "datePublished": "2024-04-25T00:00:00"
    },
    "containers": {
        "cna": {
            "providerMetadata": {
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre",
                "dateUpdated": "2024-09-03T19:34:19.749577"
            },
            "descriptions": [
                {
                    "lang": "en",
                    "value": "python-jose through 3.3.0 has algorithm confusion with OpenSSH ECDSA keys and other key formats. This is similar to CVE-2022-29217."
                }
            ],
            "affected": [
                {
                    "vendor": "n/a",
                    "product": "n/a",
                    "versions": [
                        {
                            "version": "n/a",
                            "status": "affected"
                        }
                    ]
                }
            ],
            "references": [
                {
                    "url": "https://github.com/mpdavis/python-jose/issues/346"
                },
                {
                    "url": "https://www.vicarius.io/vsociety/posts/algorithm-confusion-in-python-jose-cve-2024-33663"
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "type": "text",
                            "lang": "en",
                            "description": "n/a"
                        }
                    ]
                }
            ]
        },
        "adp": [
            {
                "problemTypes": [
                    {
                        "descriptions": [
                            {
                                "type": "CWE",
                                "cweId": "CWE-327",
                                "lang": "en",
                                "description": "CWE-327 Use of a Broken or Risky Cryptographic Algorithm"
                            }
                        ]
                    }
                ],
                "affected": [
                    {
                        "vendor": "python-jose_project",
                        "product": "python-jose",
                        "cpes": [
                            "cpe:2.3:a:python-jose_project:python-jose:*:*:*:*:*:*:*:*"
                        ],
                        "defaultStatus": "unknown",
                        "versions": [
                            {
                                "version": "0",
                                "status": "affected",
                                "lessThanOrEqual": "3.3.0",
                                "versionType": "custom"
                            }
                        ]
                    }
                ],
                "metrics": [
                    {
                        "cvssV3_1": {
                            "scope": "UNCHANGED",
                            "version": "3.1",
                            "baseScore": 6.5,
                            "attackVector": "NETWORK",
                            "baseSeverity": "MEDIUM",
                            "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:N",
                            "integrityImpact": "LOW",
                            "userInteraction": "NONE",
                            "attackComplexity": "LOW",
                            "availabilityImpact": "NONE",
                            "privilegesRequired": "NONE",
                            "confidentialityImpact": "LOW"
                        }
                    },
                    {
                        "other": {
                            "type": "ssvc",
                            "content": {
                                "timestamp": "2024-07-17T13:37:11.286073Z",
                                "id": "CVE-2024-33663",
                                "options": [
                                    {
                                        "Exploitation": "poc"
                                    },
                                    {
                                        "Automatable": "no"
                                    },
                                    {
                                        "Technical Impact": "partial"
                                    }
                                ],
                                "role": "CISA Coordinator",
                                "version": "2.0.3"
                            }
                        }
                    }
                ],
                "title": "CISA ADP Vulnrichment",
                "providerMetadata": {
                    "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
                    "shortName": "CISA-ADP",
                    "dateUpdated": "2024-07-17T13:45:02.277Z"
                }
            },
            {
                "providerMetadata": {
                    "orgId": "af854a3a-2127-422b-91ae-364da2661108",
                    "shortName": "CVE",
                    "dateUpdated": "2024-08-19T07:38:26.262Z"
                },
                "title": "CVE Program Container",
                "references": [
                    {
                        "url": "https://github.com/mpdavis/python-jose/issues/346",
                        "tags": [
                            "x_transferred"
                        ]
                    },
                    {
                        "url": "https://www.vicarius.io/vsociety/posts/algorithm-confusion-in-python-jose-cve-2024-33663"
                    }
                ]
            }
        ]
    },
    "dataVersion": "5.1"
}